conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/NegationRuleSuite.scala

```diff
@@ -39,14 +39,19 @@ class NegationRuleSuite extends FunSuite with SparkTestBase with LoggerTestBase{
 
     assert(intercept[ValidationException] {
       NegationRuleInterpreter.validateInputField("dataset", schema, "date")
-    }.getMessage contains "field is not a numeric type")
+    }.getMessage contains "field is not a algebraic type")
```

Comment:
Just a reminder. If you going to change the error message of the exception do not forget to change it here as well.

Reply:
I can promise you, I would forget. Thanks